Source/qol/xpbar.cpp

```diff
@@ -68,7 +68,7 @@ void DrawXPBar(const CelOutputBuffer &out)
 	if (!sgOptions.Gameplay.bExperienceBar)
 		return;
 
-	const PlayerStruct &player = plr[myplr];
+	const auto &player = plr[myplr];

```

Comment:
I noticed some other places where `PlayerStruct` is used instead of `auto`. Is this supposed to be `auto` in all places?

Reply:
Yes, I did a search so i think i fixed all of them.